src/csvwidget/widget.ts

```diff
@@ -42,6 +59,18 @@ class CSVWidget extends Widget {
     this.node.tabIndex = -1;
     this.addClass(CSV_CLASS);
 
+    this.layout = new PanelLayout();
+    this._toolbar = new Widget({ node: createDelimiterSwitcherNode() });
+    this._toolbar.addClass(CSV_TOOLBAR_CLASS);
+    this._table = new Widget();
+    this._table.addClass(CSV_TABLE_CLASS);
+
+    let layout = this.layout as PanelLayout;
+    layout.addWidget(this._toolbar);
+    layout.addWidget(this._table);
+
+    let select = this._toolbar.node.firstChild as HTMLSelectElement;
```

Comment:
The `firstChild` is the `span`, it would be safer to use `this._toolbar.node.getElementsByClassName('jp-CSVToolbarDropdown')[0]`